lib/fiware-pep-steelskin.js

```diff
@@ -406,7 +406,7 @@ function startProxy(callback) {
  *                               field of the returned object, but the whole object).
  */
 function stopProxy(proxy, callback) {
-    logger.info('Creating proxy');
+    logger.info('Closing proxy');
```

Comment:
Maybe better

```suggestion
    logger.info('Stopping proxy');
```

?